internal/controllers/core/podlogstream/podlogstreamcontroller.go

```diff
@@ -433,9 +435,16 @@ func (r *Controller) updateStatus(streamName types.NamespacedName) {
 		return
 	}
 
+	lastUpdate, hasLastUpdate := r.lastUpdate[streamName]
+	if hasLastUpdate && cmp.Equal(status, lastUpdate) {
```

Comment:
I realize this is actually the same as before, but any reason to use `cmp` vs the K8s `equality.Semantic.DeepEqual()`?

Reply:
done